.github/workflows/check_format.yml

```diff
@@ -0,0 +1,28 @@
+name: Check format
```

Comment:
Maybe before this line, we can add the authorship of the code 😄 similar to what happens in the Snitch cluster. Something like:

```
# ---------------------------------------------------------
# Copyright 2023 KULeuven
# Author: Xiaoling Yi <xiaoling.yi@esat.kuleuven.be>
# ---------------------------------------------------------
```

Reply:
Maybe for all the other files?

Reply:
ok!